src/ServiceControl.Monitoring/MetricReport.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus.Metrics
+{
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// The reporting message.
+    /// </summary>
+    public class MetricReport : ICommand
```

Comment:
I am not sure what you are trying to achieve here. NServiceBus.Metrics depends on Metrics.NET and uses it to generate Json. We will use Metrics.NET here internally to generate derived metrics (most likely). What would we gain from pretending we don't use Metrics.NET?